### ISSUE TITLE
Missing Github notifications

### ISSUE THREAD

**Issue by Munter** [greenkeeperio/greenkeeper#31]
When Greenkeeper open new PR's it does so on behalf of me. This means I never get any Github notifications about it. So now I manually have to walk through all my projects to go looking for them instead of getting the direct access.

Is there a way to trigger notifications for me?

**Comment by AdriVanHoudt** [comment]
Same problem here, right now the emails work but this is not ideal

**Comment by boennemann** [comment]
The obvious way to send you notifications would be to create pull-requests with a Greenkeeper account. Unfortunately that isn't possible, because it is against the API terms to send automated PRs from another account.

I know that e.g. gitter is sending automated PRs from their account, but we don't want to put our service at risk of being shut down. Also we'll soon be sending several thousand PRs a day, which, in contrast to gitter, will bring us to the limits of a single account's rate limit very quickly. On top of that we think it is a plus to have dedicated branches for every PR, so that you can just pick it up and add fixes if you need to. We can't create those from another account.

We'd love to have some way of sending you the PR in another way, e.g. a delegate content creation API by GitHub, but such a thing is currently not available. We're trying to figure out a solution with GitHub, but for now this how we have to do it – even though we don't like to do it this way.

One way to get GitHub built-in emails would be to just signup with a bot account and reenabling the repo from there. We've added a config flag to disable our emails. (Add `"greenkeeper": {"emails": false}` to your package.json)

Also regarding emails. It seems like you didn't received those. a) Please check your spam folder b) We've now added the correct DKIM record, so they should arrive now.

Thanks for understanding,

Stephan

**Comment by AdriVanHoudt** [comment]
That is indeed not ideal, cool that you are talking to github about it

**Comment by dbkaplun** [comment]
+1 please fix!

**Comment by boennemann** [comment]
@dbkaplun We're on it. Awaiting GitHub reply :)

**Comment by Munter** [comment]
Well, at least https://github.com/pulls gives sort of a decent overview. But you still have to check it manually

**Comment by boennemann** [comment]
Good news everyone, we'll soon be able to open PRs from our bot account.
I'll let you know once that lands.

**Comment by Munter** [comment]
:dancers:

**Comment by AdriVanHoudt** [comment]
:clap:

**Comment by boennemann** [comment]
It's live now. Thanks y'all for your great feedback and patience.

:tada:

**Comment by boennemann** [comment]
This is how we're doing it now:

We will still create branches from users' accounts, but only the actual pull request will be opened by our bot account. We talked to GitHub and they said this is acceptable use of the API, because this is something y'all opt-in to. Caveat: This only works for public repos.

We can run on this strategy until we have 5000 pull requests per hour. We will delay all pull requests until we have rate limit resources again. Should we reach this point we will talk to GitHub so we can get an increased rate limit.

**Comment by AdriVanHoudt** [comment]
Turns out that having slack really helps :D but this cool, are you guys close to 5000 PR's/hour? That would be crazy!

**Comment by boennemann** [comment]
@AdriVanHoudt No we currently aren't, but imagine we're getting a mit more spread and e.g. lodash releases a new version. This is something we could easily reach in peak times ;)

**Comment by AdriVanHoudt** [comment]
ah yeah can imagine you have a lot of peaks, I see with my repos, they use some of the same deps, that when there is a new release I get instant 3 PR's, which is cool
